lab2.js

```diff
@@ -114,10 +155,17 @@ assert((new Human()).sayHello(new Klingon()) === "nuqneH",
 //*********************************************************
 function max(array) {
   // TODO: return the largest number in the given array
+  array.sort(function(a, b) {
```

Comment:
Nice! Way to use JavaScript built-ins and callbacks to make your life easy. The only downside of this approach is that you've mutated the original array (by sorting it), which could be a significant side effect.